aip/0166.md

```diff
@@ -0,0 +1,78 @@
+---
+aip:
+  id: 166
+  state: reviewing
+  created: 2020-06-24
+permalink: /166
+redirect_from:
+  - /0166
+---
+
+# Hierarchically-derived resources
+
+Services occasionally have resources where the resource's effective values are
+determined by consulting both configuration on the resource as well as
```

Comment:
The sentence captures direct policies (which are "on" a resource or parent), but I don't think it clues folks into the fact that it also includes shared policies which are _bound_ to the resource or parent.

Reply:
I am not sure what "bound" means here.